src/routes/search/+page.server.ts

```diff
@@ -0,0 +1,8 @@
+import type { Actions } from './$types';
```

Comment:
you can remove this file, we will use static site generation thus dynamic server files won't work. The search is not under this issue scope.